docs/admin/deployment/tor_pow.rst

```diff
@@ -8,7 +8,7 @@ Tor network as a whole <https://blog.torproject.org/tor-network-ddos-attack/>`_
 as well as burden individual onion services, including SecureDrops.
 
 Tor now includes a `proof-of-work (PoW) defense
-<https://community.torproject.org/onion-services/ecosystem/technology/pow/>`_
+<https://onionservices.torproject.org/technology/pow/#general-questions>`_
```

Comment:
For the record, this is the same page, last seen at <https://web.archive.org/web/20240621020201/https://community.torproject.org/onion-services/ecosystem/technology/pow/>.